src/api/app/models/user_ldap_strategy.rb

```diff
@@ -37,7 +37,7 @@ def self.find_group_with_ldap(group)
   end
 
   # This static method performs the search with the given search_base, filter
-  def self.search_ldap(search_base, filter, required_attr = nil)
```

Comment:
Can you please explain why this attribute is unused and write it also in the commit message?